test/integration/api_v1_test.rb

```diff
@@ -586,6 +586,27 @@ class ApiV1Test < ActionDispatch::IntegrationTest
     assert_response :unauthorized
   end
 
+  test 'add a key to a project' do
```

Comment:
@jaypoulz Is there a list somewhere of other users I can use for tests? The only user I could find in this test file is kcarcia@cs.uml.edu and I am not sure where that user lives.